requirements.txt

```diff
@@ -16,7 +16,7 @@ pluggy==0.13.1
 pockets==0.9.1
 py==1.10.0
 Pygments==2.9.0
-pyinotify==0.9.6
+# pyinotify==0.9.6
```

Comment:
Feel free to completely delete the line, I'll need to re-do the requirements anyway before releasing a version update :)

Reply:
Thank you.